babel/src/test/resources/sql/big-query.iq

```diff
@@ -668,6 +668,77 @@ SELECT SAFE_ADD(CAST('NaN' AS DOUBLE), CAST(3 as BIGINT)) as NaN_result;
 (1 row)
 
 !ok
+
+#####################################################################
+# SAFE_DIVIDE
```

Comment:
Yeah I figured there implementations were similar enough that it might be a bit tedious for reviewers to review two very similar PRs, but I'm happy to split it up if it's easier. What do you think?

Reply:
Two separate PRs look better to me, it would make things more clear.